packages/ics721/src/execute.rs

```diff
@@ -513,15 +524,23 @@ where
             msg: to_json_binary(&ExecuteMsg::Callback(CallbackMsg::Mint {
                 class_id: class.id.clone(),
                 receiver,
-                tokens,
+                tokens: tokens.clone(),
             }))?,
             funds: vec![],
         };
 
-        let instantiate = self.create_instantiate_msg(deps, &env, class.clone())?;
+        let (class_id_info, instantiate) =
+            self.create_instantiate_msg(deps, &env, class.clone())?;
+
+        let token_ids = format!("{:?}", tokens);
+        let event = Event::new("ics721_receive_create_vouchers")
+            .add_attribute("class_id", class_id_info.class_id)
+            .add_attribute("nft_contract", class_id_info.address)
+            .add_attribute("token_ids", token_ids);
```

Comment:
Also added 3 new events, for better processing (e.g. for constellations):

1. event when NFTs are created/minted...